[2025-09-21T11:29:05.755+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:29:05.756+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:29:05.812+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:29:05.814+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:29:05.915+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:28:00+00:00 [queued]>
[2025-09-21T11:29:05.928+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:28:00+00:00 [queued]>
[2025-09-21T11:29:05.929+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T11:29:05.955+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): trigger_model_training> on 2025-09-21 11:28:00+00:00
[2025-09-21T11:29:05.967+0000] {standard_task_runner.py:57} INFO - Started process 938 to run task
[2025-09-21T11:29:05.973+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'trigger_model_training', 'scheduled__2025-09-21T11:28:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmps3qcbyg9']
[2025-09-21T11:29:05.978+0000] {standard_task_runner.py:85} INFO - Job 76: Subtask trigger_model_training
[2025-09-21T11:29:06.049+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:29:06.050+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:28:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:29:06.052+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:28:00+00:00 [running]> on host 36da5371e73d
[2025-09-21T11:29:06.178+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='trigger_model_training' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T11:28:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T11:28:00+00:00'
[2025-09-21T11:29:10.929+0000] {python.py:194} INFO - Done. Returned value was: d315743a7b544f48b09d762fe403f136
[2025-09-21T11:29:11.041+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=trigger_model_training, execution_date=20250921T112800, start_date=20250921T112905, end_date=20250921T112911
[2025-09-21T11:29:11.214+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T11:29:11.244+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
